### PR TITLE
Feature/likes api

### PR DESCRIPTION
**Pull request by fredford**

### CODE REVIEW

**Review by timvm1108 — APPROVED** [review]
lgtm

File: manager/backend/api.py [hunk 1]
```diff
@@ -198,52 +197,128 @@ def destroy(self, request, author_id=None, id=None, *args, **kwargs):
...
+			description = request.data["description"],
+			content_type = request.data["contentType"],
+			content = request.data["content"],
+			categories = request.data["categories"],
+			count = 0,
+			visibility = request.data["visibility"],
+			unlisted = request.data["unlisted"]
+		)
+		elif author_id:
+			post = Post(
```
Comment (kangxi11) [inline manager/backend/api.py:219]: what's the difference between this and the above if code block? Can we shorten it to just `if author_id`

File: manager/backend/api.py [hunk 2]
```diff
@@ -198,52 +197,128 @@ def destroy(self, request, author_id=None, id=None, *args, **kwargs):
 			# Return the serializer output data as the response
 			return Response(deleted_post)
 		return super().destroy(request, *args, **kwargs)
+		
+	def create(self, request, author_id=None, id=None, *args, **kwargs):
+		if author_id and id:
+			post = Post(
+			author = Author.objects.filter(id=author_id).get(),
```
Comment (kangxi11) [inline manager/backend/api.py]: can we indent these since they are arguments for `Post()`

File: manager/frontend/src/containers/Login.js [hunk 3]
```diff
@@ -1,122 +1,135 @@
-import React, { useState } from 'react';
-import { Link } from 'react-router';
-import { connect } from 'react-redux';
+import React, { useState } from "react";
```
Comment (kangxi11) [inline manager/frontend/src/containers/Login.js:1]: can you remove `Login.js` from your merge by pulling from master

**Pull request merged**